app/dashboard/organization/overview/page.tsx

```diff
@@ -133,6 +133,7 @@ export default function OrganizationOverview() {
             title: "Successfully Left Organization",
             })
             router.push("/dashboard/organization/overview/orgSignUp");
+            router.refresh()
```

Comment:
This is fixed but there it looks like a "double refresh" since I had to refresh the page to force update the sidenav and then redirect user to page. I can't find a way to refresh on redirect. It works fine it just looks like the page refreshes twice on user POV

Reply:
It shows as a single page refresh for me, unless you're counting when the page has to render the "Create an Organization" section as a refresh too.